qt-build/Cargo.toml

```diff
@@ -0,0 +1,19 @@
+# SPDX-FileCopyrightText: 2022 Klarälvdalens Datakonsult AB, a KDAB Group company <info@kdab.com>
+# SPDX-FileContributor: Be Wilson <be.wilson@kdab.com>
+#
+# SPDX-License-Identifier: MIT OR Apache-2.0
+
+[package]
+name = "qt-build"
```

Comment:
Once this is stablish API wise, wonder if this should be added to it's own repository ? Eg `qt-build-rs` like we did with `clang-format-rs` etc ?

Reply:
I don't mind either way.

Reply:
FWIW I set the initial version of qt-build to 0.3 to match the rest of the cxx-qt crates, but if we want to move it out of this repo, maybe it should initially be published with version 0.1.

Reply:
I think for now while things are being developed lets keep it together. If it is separated out the version doesn't really matter, as it'll likely head rapidly towards 1.0 anyway :-)